source/text.tex

```diff
@@ -13317,7 +13317,7 @@
 These functions have the semantics specified in the C standard library.
 \end{itemdescr}
 
-\xrefc{7.22.7.1, 7.22.8, 7.29.6.2.1}
+\xrefc{7.24.8.1, 7.24.9, 7.31.6.3.1}
```

Comment:
```suggestion
\xrefc{7.24.8.2, 7.24.9, 7.31.6.3.1}
```
I wonder if we can just remove this entire paragraph.

Reply:
Not in this pull request.